### DISCUSSION TITLE
📅 call #wg-saft-ao: 2023/07/12

### DISCUSSION THREAD

**Discussion by cryptolopes** [assoft-portugal/SAF-T-AO#224] [Announcements]
### ⚽ Objetivo da reunião:

  

*   Apresentação e discussão de problema relacionado com a emissão de recibos com origem em notas de crédito, que têm por base IVA negativo.
*   Apresentação de novidades

  

### ⌚ Hora de início:

  

*   13.30 GMT

  

#### Como proceder à regularização de uma fatura usando o saldo da conta corrente do cliente?

Podem existir casos em que o saldo da conta corrente do cliente é favorável ao cliente. Nos casos em que isso acontece, usa-se esse saldo para regularizar as faturas vincendas, isto é, aquelas que forem realizadas após a constatação desse saldo positivo a favor do cliente.

Nos casos em que há faturas sujeitas a IVA e outras não sujeitas a IVA surge a dúvida sobre o IVA deve ou não ser considerado no momento da emissão do recibo, uma vez que podem existir casos em que a soma desse IVA é negativa.

**Veja-se o seguinte cenário**

1. Emissão de `FT C21/1711` de `1.140,00` com IVA de `140,00`
2. Emissão de `RC RCC21/573` de `1.140,00` com IVA de `140,00`
3. Emissão de `NC C21/108`   de `1.140,00` com IVA de `140,00`
4. Emissão de `FT C21/1712`  de `2.000,00` sem IVA
5. Emissão de `RC RCC21/574` referente à FT C21/1712 em que para o efeito é usado o saldo da conta corrente a favor do cliente no valor de `1.140,00` e um pagamento em numerário de `860.00`.

**Dúvida**

Ao ter em consideração o saldo de 1.140,00, que foi apurado com IVA, para a regularização (emissão de recibo) da fatura sem IVA, terá como consequência um valor de -140.00 de IVA?

**Solução**

* O IVA não deve ser considerado para a emissão do recibo de quitação da FT C21/1712.
* O IVA devido foi automaticamente regularizado no momento da emissão da NC C21/108 e por essa razão o saldo a favor do cliente é integralmente líquido.
* O saldo a favor do cliente atua como um meio de pagamento.

**Exemplos**

```xml
<!-- Recibo RC RCC21/573 -->
<!-- A inclusão do elemento Tax é opcional no caso. -->

<Payment>
    <PaymentRefNo>RC RCC21/573</PaymentRefNo>
    <TransactionDate>2023-07-11</TransactionDate>
    <PaymentType>RG</PaymentType>
    <DocumentStatus>
        <PaymentStatus>N</PaymentStatus>
        <PaymentStatusDate>2023-07-11T14:50:31</PaymentStatusDate>
        <SourceID>vi</SourceID>
        <SourcePayment>P</SourcePayment>
    </DocumentStatus>
    <PaymentMethod>
        <PaymentMechanism>TB</PaymentMechanism>
        <PaymentAmount>1140.00</PaymentAmount>
        <PaymentDate>2023-01-11</PaymentDate>
    </PaymentMethod>
    <SourceID>vi</SourceID>
    <SystemEntryDate>2023-07-11T14:50:31</SystemEntryDate>
    <CustomerID>VIC0000000000006295</CustomerID>
    <Line>
        <LineNumber>1</LineNumber>
        <SourceDocumentID>
            <OriginatingON>FT C21/1711</OriginatingON>
            <InvoiceDate>2023-07-11</InvoiceDate>
        </SourceDocumentID>
        <CreditAmount>1000.00</CreditAmount>
        <Tax>
            <TaxType>IVA</TaxType>
            <TaxCountryRegion>AO</TaxCountryRegion>
            <TaxCode>NOR</TaxCode>
            <TaxPercentage>14.00</TaxPercentage>
        </Tax>
    </Line>
    <DocumentTotals>
        <TaxPayable>140.00</TaxPayable>
        <NetTotal>1000.00</NetTotal>
        <GrossTotal>1140.00</GrossTotal>
    </DocumentTotals>
</Payment>
```

```xml
<!-- Recibo RC RCC21/574 -->

<Payment>
    <PaymentRefNo>RC RCC21/574</PaymentRefNo>
    <TransactionDate>2023-07-11</TransactionDate>
    <PaymentType>RG</PaymentType>
    <DocumentStatus>
        <PaymentStatus>N</PaymentStatus>
        <PaymentStatusDate>2023-07-11T14:53:31</PaymentStatusDate>
        <SourceID>vi</SourceID>
        <SourcePayment>P</SourcePayment>
    </DocumentStatus>
    <PaymentMethod>
        <PaymentMechanism>CS</PaymentMechanism>
        <PaymentAmount>1140.00</PaymentAmount>
        <PaymentDate>2023-07-11</PaymentDate>
    </PaymentMethod>
    <PaymentMethod>
        <PaymentMechanism>NU</PaymentMechanism>
        <PaymentAmount>860.00</PaymentAmount>
        <PaymentDate>2023-07-11</PaymentDate>
    </PaymentMethod>
    <SourceID>vi</SourceID>
    <SystemEntryDate>2023-07-11T14:53:31</SystemEntryDate>
    <CustomerID>VIC0000000000006295</CustomerID>
    <Line>
        <LineNumber>1</LineNumber>
        <SourceDocumentID>
            <OriginatingON>FT FTC21/1712</OriginatingON>
            <InvoiceDate>2023-07-11</InvoiceDate>
        </SourceDocumentID>
        <CreditAmount>2000.00</CreditAmount>
    </Line>
    <DocumentTotals>
        <TaxPayable>0.00</TaxPayable>
        <NetTotal>2000.00</NetTotal>
        <GrossTotal>2000.00</GrossTotal>
    </DocumentTotals>
</Payment>
```

  

  

  

### 📝 Novidades

  

*   Vão existir novidades e alterações legistivas com implicações nas regras da validação de software.
*   Os contribuintes vão ter que identificar as aplicações em uso junto da AGT.
*   Vamos dar início a uma fase de estudo do SAF-T (AO) da Contabilidade.

  

### ⏭️ Próxima reunião:

  

2023-07-26 14.30

  

### 📁 SAF-T AO

  

[![](https://img.shields.io/github/downloads/assoft-portugal/SAF-T-AO/total)](https://github.com/assoft-portugal/SAF-T-AO/releases) [![](https://img.shields.io/github/issues-raw/assoft-portugal/SAF-T-AO)](https://github.com/assoft-portugal/SAF-T-AO/issues) [![](https://img.shields.io/github/v/tag/assoft-portugal/SAF-T-AO)](https://github.com/assoft-portugal/SAF-T-AO/releases) [![](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/assoft-portugal/SAF-T-AO/blob/master/LICENSE) [![](https://img.shields.io/badge/discusion-telegram-blue)](https://t.me/saftao)

**Comment by patricialmeida** [comment]
Boa tarde
Penso que temos de clarificar melhor esta situação pois:
- No meu entendimento o exemplo indicado seria de no recibo exportado para o saft indicar o pagamento integral da fatura FT C21/1712 (2000 AOA), incluindo no mesmo recibo a regularização da nota de credito NC C21/108 (1140 AOA), sendo que o valor regularizado seria a diferença entre o valor dos dois documentos
- Já sabemos que isto iria originar um valor de regularização de IVA negativo e que a estrutura do SAFT não permite
- Uma solução seria regularizar parcialmente a fatura como a proposta feita e fazer outro recebimento sem emissão de recibo que apenas regularizava a conta corrente desses documentos com saldo zero
- Nesta solução a minha questão é apenas se **é legal em Angola**, dado a obrigatoriedade de emissão do recibo sobre todos os documentos regularizados, emitir recibos sobre documentos de naturezas diferentes que se saldem entre si, como neste exemplo apresentado, ou se é possível apenas fazer o movimento a nível contabilístico sem a emissão do recibo, dado que efetivamente o valor recebido é zero.
